### DISCUSSION TITLE
Missing contacts

### DISCUSSION THREAD

**Discussion by WebworkrNet** [FossifyOrg/General-Discussion#661] [Q&A]
**Device**: Samsung Galaxy A13 5G (SM-A136B/DSN)
**Operating System**: Android 14, One UI-Version 6.0
**Fossify Contacts**: 1.0.1
Parallel to Fossify, I currently have the Samsung Contacts app installed so that I can make a comparison. I have noticed that some contacts sometimes appear under Samsung, but not under Fossify.

- "Merge duplicate contacts" is switched off.
- All accounts set up on the device are shown.

What could be the cause?

Thank you

**Comment by WebworkrNet** [comment]
Maybe related to issue **Contacts load with a long delay or not at all** https://github.com/FossifyOrg/General-Discussion/issues/220.

**Comment by naveensingh** [comment]
> What could be the cause?

Hard to say based on the provided info. 

It could be the same issue as https://github.com/FossifyOrg/Contacts/issues/261. Feel free to file a bug report if you are still facing this issue.
